app/src/main/java/io/seqera/wave/cli/App.java

```diff
@@ -374,6 +377,10 @@ protected void validateArgs() {
         if( !isEmpty(contextDir) && isEmpty(containerFile) )
             throw new IllegalCliArgumentException("Option --context requires the use of a container file");
 
+        // Check imageName
+        if( isEmpty(imageName) )
```

Comment:
This syntax causes the app to crash every time no image name is provided as input (see failed unit tests), as imageName is rightly empty by default. 

The old syntax, prior to https://github.com/seqeralabs/wave-cli/pull/57/commits/6bca65cd362f436112e36fbb21b86fa8a32eabce, would throw an exception in the case `--image-name "   "`, which I think is quite unusual as an input typo.

You can probably avoid this check entirely.

Reply:
done